start-from-zero/sass-architecture/sass/abstract/_media-queries.sass

```diff
@@ -0,0 +1,104 @@
+$small-font: 12px
+@media (max-width: 800px)
+    .column-30
+        width: 48%
+        margin-bottom: 10px
+        
+    .column-30:nth-of-type(3)
+        
+    #table-row th 
+        font-size: $small-font
+        
+    #tbody
+        tr
+            td
+                font-size: $small-font
+          
+    .full-width
+        width: 100%
+    
+    .margin-right
+        margin-right: 1%
+        
+    option, select, .description
+        font-size: $small-font
+        
+    
+        
+    
+        
+@media (max-width: 600px)
+    .column-30
+        width: 100%
+        
+    .margin-right
+        margin-right: 0
+        
+        
+    
+    nav

```

Comment:
I don't like the heavy nesting since it increases very much the specificity of the selectors without being necessary. Please focus on fixing these architectural issues with high priority. I would like to almost entirely simple, well named classes without any nesting whatsovever